.vscode/settings.json

```diff
@@ -23,5 +23,6 @@
   "git.ignoreLimitWarning": true,
   "cSpell.words": [
     "datalake"
-  ]
+  ],
+  "typescript.tsdk": "node_modules\\typescript\\lib"
```

Comment:
Was this intentional?

Reply:
No, good call. It was a change I had to make to my VS Code workspace